### PR TITLE
Mob desk popup

### PR DESCRIPTION
**Pull request by eudondian**
- Popup  details  project appears when the user clicks (or taps) the button to check project details on the mobile
- Popup disappears when the user clicks (or taps) the close (X) button.
- The same interaction is applicable to the desktop version

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -384,118 +322,295 @@ textarea {
...
+  top: 50%;
+  left: 45%;
+  transform: translate(-50%, -50%) scale(0);
+  background: white;
+  width: 80%;
+  height: 100%;
+  z-index: 2;
+  text-align: justify;
+  padding: 20px;
+}
```
Comment (vechicin) [inline style.css]: Your pop-up window doesn't follow the template design on the desktop version.

Your design:

Screen Shot 2022-06-24 at 8 21 47

The template design:

Screen Shot 2022-06-24 at 8 22 30

Please fix this.

File: index.html [hunk 2]
```diff
@@ -1,309 +1,500 @@
...
+    <section id="wrapper">
+      <!--Header Section -->
+      <!-- <section id="header">
+            <nav class="nav">
+                <h3 class="logo">My Logo</h3>
+                <div class="menu-icon">
+                    <i class="uil uil-bars"></i>
+                </div>
+            </nav>
+        </section> -->
```
Comment (vechicin) [inline index.html]: Please remove this commented code to keep a cleaner codebase.

File: index.html [hunk 3]
```diff
@@ -1,309 +1,500 @@
...
+          </li>
+          <li>
+            <a href=""><i class="fa fa-angellist"></i></a>
+          </li>
+        </ul>
+      </section>
     </section>
 
     <!-- Works Section -->
     <section id="cards">
```
Comment (vechicin) [inline index.html]: The content of this section needs to be dynamically generated.

Screen Shot 2022-06-24 at 8 25 28

Please do so.

File: script/index.js [hunk 4]
```diff
@@ -1,16 +1,40 @@
...
+
+cardBtn.forEach((item) => {
+  item.addEventListener('click', () => {
+    document.querySelector('#popup-1').classList.toggle('active');
+  });
 });
+
+popBtn.addEventListener('click', () => {
+  popUp.style.display = 'none';
+});
```
Comment (vechicin) [inline script/index.js]: I can only open the modal window once after page reload. Every subsequent attempt at opening a mobile window throws an error in the console. The reason for that is that you are applying an `onclick` attribute directly to the HTML calling a `popup()` method that becomes null after the first click. You should have an array or an object that holds the section information and create a function that opens and closes the modal window for each project.

Screen Shot 2022-06-24 at 8 29 00

Please fix this.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @eudondian and @krisbobo ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @eudondian 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**